lib/ruby_libstorj/env.rb

```diff
@@ -13,8 +13,12 @@ class Env
     end
 
     def initialize(*options)
-      @pointer = ::LibStorj::Ext::Storj.method(:init_env).call(*options)
-      @pointer[:loop] = ::Libuv::Ext.default_loop
+      @storj_env = ::LibStorj::Ext::Storj.method(:init_env).call(*options)
+      @storj_env[:loop] = ::Libuv::Ext.default_loop
```

Comment:
u think that is going to cause issues? everything in libstorj references off of `env->loop` right?